keyboards/sauceball/readme.md

```diff
@@ -0,0 +1,28 @@
+# sauceball
+
+![sauceball DIY trackball](https://i.imgur.com/h5vI6Rqh.jpg)
+
+*A trackball design for recycling old PS/2 compatible mice into trackballs.*
+
+* Keyboard Maintainer: [xia0](https://github.com/xia0)
+* Hardware Supported: Atmega32u4
+* Hardware Availability: [github](https://github.com/xia0/sauceball_files)
+    
+
+Make example for this keyboard (after setting up your build environment):
+
+    make sauceball:default
+
+Flashing example for this keyboard:
+
+    make sauceball:default:flash
+
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
+
+## Bootloader
+
+Enter the bootloader in 3 ways:
+
+* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
+* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
```

Comment:
Can you update this to actually reflect your hardware please, is there actually a reset button?

Reply:
The default message already seems pretty accurate. The Teensy 2.0 has a reset button but the Arduino Pro Micros have pads that need to be shorted.